packages/documentation-framework/templates/mdx.js

```diff
@@ -90,28 +83,27 @@ const MDXChildTemplate = ({
           {`This page showcases templates for the ${id.toLowerCase()} component. A template combines a component with logic that supports a specific use case, with a streamlined API that offers additional, limited customization.`}
         </InlineAlert>
       )}
-    </React.Fragment>
+    </StackItem>
   );
   // Create dynamic component for @reach/router
   const ChildComponent = () => (
-    <div className="pf-v6-u-display-flex ws-mdx-child-template">
+    <div className={source !== 'landing-pages' ? 'pf-v6-l-flex' : ''}>
       {toc.length > 1 && (
         <TableOfContents items={toc} />
       )}
-      <div>
-        <div className={innerContentWrapperClass()}>
-          {InlineAlerts}
+      <div className={isComponentCodeDocs && 'pf-v6-l-stack pf-m-gutter'} style={{...(source !== 'landing-pages' && {maxWidth: "825px"})}}>
```

Comment:
I'm assuming you can't use `<Stack>` here easily because of the conditional classes/styles, but it would be great to not have the classes hardcoded 🥸👍